script/Arcueid/Day 7/07_00_ARC07_3_1.txt

```diff
@@ -110,7 +110,7 @@ Geez, what kind of person do you take me for?! Are you misunderstanding somethin
 [sha:cf209a990933d095c5dc3326c5e9107a9aa36d8b]{
 -- Page 507, Offset 17831.
 -- 「大丈夫、そこは完璧、ブレてない。アルクェイドの事は、手加減のできないやりすぎ吸血鬼だと思ってる」
-"Don't worry, I know exactly who you are. You're a vampire who goes too far and doesn't know how to hold back."
+"Don't worry, I know exactly what to take you for. A vampire who goes too far and doesn't know how to hold back."
```

Comment:
Not sure that this sees common usage, but I do appreciate the mirroring. On the fence, would be interested in @Avuxo's opinion

Reply:
this isn't necessarily a common usage but I think I like it tbqh. I say let's go with it.